### PR TITLE
Revise tech docs in README to favour GOV.UK Docker

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh

This follows the pattern established in [1].

[1]: alphagov/content-publisher#2257

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,19 @@
 # Travel Advice Publisher
 
-Travel Advice Publisher manages foreign travel advice content on GOV.UK.
-
-## Screenshots
-
-![Travel Advice Publisher](docs/images/screenshot.png)
```
Comment (ChrisBAshton) [inline README.md:7]: If we're not using the screenshot, can we delete this?

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
(This PR still needs to remove startup.sh)

File: README.md (hunk 1 above)
Comment (benthorner) [reply README.md:7]: Oops, thanks for spotting. I've amended 👍.

**Comment by benthorner** [comment]
Thanks for spotting the pending deletions @ChrisBAshton 💯. 

I've amended with both of them, so this should be ready to go.

**Review by ChrisBAshton — APPROVED** [review]
🎉

**Pull request merged**